chaoshub/publish.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+import requests
+from logzero import logger
+
+__all__ = ["publish_to_hub"]
+
+
+def publish_to_hub(hub_base_url: str, token: str, organisation: str,
+                   workspace: str, journal_path: str, journal: dict):
+    if hub_base_url and token:
```

Comment:
would be nice to have a `logger.debug(...)` statement when that conditional is False.

Reply:
Done.